tools/webdriver/webdriver/transport.py

```diff
@@ -27,17 +29,12 @@ def from_http_response(cls, http_response):
         # >       "application/json; charset=utf-8"
         # >    "cache-control"
         # >       "no-cache"
-        assert http_response.getheader("Content-Type") == "application/json; charset=utf-8"
-        assert http_response.getheader("Cache-Control") == "no-cache"
```

Comment:
Because asserting here for something that should simply be a test in the testsuite is wrong. It doesn't help anyone to ERROR on every test just because the headers are wrong, even if the implementation otherwise works.